src/rgw/rgw_log.cc

```diff
@@ -200,25 +200,21 @@ static void log_usage(req_state *s, const string& op_name)
   if (!usage_logger)
     return;
 
-  std::string user;
+  std::string user = to_string(s->owner.id);
   std::string payer;
-  string bucket_name;
-
-  bucket_name = s->bucket_name;
+  string bucket_name = s->bucket_name;
 
   if (!bucket_name.empty()) {
-    bucket_name = s->bucket_name;
     user = to_string(s->bucket_owner.id);
+
     if (!rgw::sal::Bucket::empty(s->bucket.get()) &&
-	s->bucket->get_info().requester_pays) {
+        s->bucket->get_info().requester_pays &&
+        s->err.http_ret != 403) {
```

Comment:
ok, aws docs have specific handling for 403s; could you please add a code comment that describes this?